src/Configuration/Programmer/Programmer.cpp

```diff
@@ -200,57 +218,45 @@ void programmer_entry(CFGCommon_ARG* cmdarg) {
     int status = 0;
     InitLibrary(openOcdExecPath.string());
```

Comment:
Do we still need to init openocd library? This should be handled in openocd adapter.

Reply:
this API setup the openocdExecPath whenever user want to use the C style programmer API such as I `ListDevices`, `ProgramFpga`, `ProgramOtp`.